modules/overlays/custom-packages.nix

```diff
@@ -37,19 +37,16 @@
         # First, weston package is overridden (passing colord = null)
         (prev.weston.override {
           colord = null;
+
+          pipewire = null;
+          freerdp = null;
+          xwayland = null;
```

Comment:
This will break legacy X apps - which I think is fine in the long run considering the security of Ghaf GUI stack to be isolated in the GUIVM. If/when legacy X apps are used with ghaf, those will go outside GUIVM to AppVMs of their own and must then include their own xwayland. Whether remote xwayland will work cross-domain (from AppVM with X app to GUIVM) is another story and trial to be tested beyond this PR.